yt/frontends/cholla/data_structures.py

```diff
@@ -111,7 +112,10 @@ def _parse_parameter_file(self):
             self.current_time = attrs["t"][:]
             self._periodicity = tuple(attrs.get("periodicity", (False, False, False)))
             self.gamma = attrs.get("gamma", 5.0 / 3.0)
-            self.mu = attrs.get("mu", 1.0)
+            with suppress(KeyError):
```

Comment:
Why not simply 

```suggestion
            if "mu" in attrs:
```
?

Reply:
Obviously that's better! I just fixed it!

Originally, I tried to write a try...except case and then I tried to make it shorter. I clearly wasn't thinking at all